charmcraft.yaml

```diff
@@ -64,12 +64,21 @@ bases:
 
 parts:
   charm:
+    source: .
+    plugin: charm
+    charm-requirements:
+      - requirements.txt
```

Comment:
While `requirements.txt` was removed in this PR, it is dynamically created when running `charmcraft pack` in the `override-build` step.

I think it is still packing because it is the default value, so I will remove it anyway.